x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/QueryPlan.java

```diff
@@ -48,11 +48,15 @@ public AttributeSet outputSet() {
 
     public AttributeSet inputSet() {
         if (lazyInputSet == null) {
-            List<Attribute> attrs = new ArrayList<>();
+            int size = 0;
```

Comment:
Could we just use `child().outputSet()` for unary plans?

Reply:
This was added recently in https://github.com/elastic/elasticsearch/pull/125938/files#diff-e3b5f4b0bc4d69a71110d01a2e93bc0813472d7ebec8472d837e7c37806ff884